### PR TITLE
Replaced Third Party pull request action with Github CLI

### PR DESCRIPTION
**Pull request by Monstarrrr**

### CODE REVIEW

File: .github/workflows/decision.yml [hunk 1]
```diff
@@ -82,13 +83,7 @@ jobs:
...
-          token: ${{ secrets.GITHUB_TOKEN }}
-          commit-message: 'Add documentation for decision: ${{ github.event.issue.title }}'
-          title: 'Automated PR ${{ github.event.issue.title }}'
-          labels: 'documentation,decision'
-          body: 'This PR was automatically created to document ${{ github.event.issue.number }}.'
-          base: 'main'
-          branch: 'docs/${{ github.event.issue.number }}_decision-${{ steps.new_branch.outputs.subject }}'
-          draft: true
+        run: |
+          gh pr create --base main --head 'docs/${ISSUE_NUMBER}_decision-${{ steps.new_branch.outputs.subject }}' --draft --label decision --label documentation --title "Add documentation for decision: ${ISSUE_TITLE}" --body "Created by Github action from ${ISSUE_NUMBER}"
```
Comment (seporterfield) [inline .github/workflows/decision.yml:87]: Needs double quotes instead of single quotes or else the shell will interpret $ literally

Comment (seporterfield) [inline .github/workflows/decision.yml:87]: GitHub cli = goated

**Review by seporterfield — CHANGES_REQUESTED** [review]

File: .github/workflows/decision.yml (hunk 1 above)
Comment (seporterfield) [inline .github/workflows/decision.yml:87]: Use $subject instead of ${{ ...subject }} since this is a line of a script and not an input for an action

**Review by seporterfield — CHANGES_REQUESTED** [review]

File: .github/workflows/decision.yml (hunk 1 above)
Comment (Monstarrrr) [reply .github/workflows/decision.yml]: Are you sure? On which variable specifically? I took example on the rest of the file which uses single brackets in these type of situation

Comment (Monstarrrr) [reply .github/workflows/decision.yml]: Nice catch 😎

**Review by seporterfield — APPROVED** [review]

**Pull request merged**